### PR TITLE
Modify some of the errors in README.md  .

### PR DESCRIPTION
**Pull request by gnehcgnaw**

### CODE REVIEW

File: dubbo-samples-rest/README.md [hunk 1]
```diff
@@ -28,12 +28,12 @@ We can use two steps to export Rest Service in Dubbo:
 Step 1: Add dependency to the pom file in our project:
 ```
         <dependency>
-            <groupId>org.apache</groupId>
+            <groupId>com.alibaba</groupId>
```
Comment (kexianjun) [inline dubbo-samples-rest/README.md]: I think make the  groupId as a placeplaceolder is better, its value is as you explained below.

File: dubbo-samples-rest/README.md [hunk 2]
```diff
@@ -28,12 +28,12 @@ We can use two steps to export Rest Service in Dubbo:
...
             <artifactId>dubbo-rpc-rest</artifactId>
             <version>${dubbo.rpc.version}</version>
         </dependency>
 ```
-
+If you are using is [2.6.1~2.6.5] version,then groupId is com.alibaba ,else groupId is org.apache.dubbo .
```
Comment (kexianjun) [inline dubbo-samples-rest/README.md]: pls make it clear that only if dubbo's version is 2.7.0 or above, its groupId is org.apache.dubbo

Comment (gnehcgnaw) [reply dubbo-samples-rest/README.md]: Your statement is more accurate , only 2.6.1  or above exist dubb-rpc-rest ,other versions are not exist .

File: dubbo-samples-rest/README.md (hunk 1 above)
Comment (gnehcgnaw) [reply dubbo-samples-rest/README.md]: I have made changes according to your suggestion.

File: dubbo-samples-rest/README.md (hunk 2 above)
Comment (gnehcgnaw) [reply dubbo-samples-rest/README.md]: I have made changes according to your suggestion.

**Comment by kexianjun** [comment]
LTGM

**Pull request merged**